sag_py_auth_brand/__init__.py

```diff
@@ -1,5 +1,5 @@
 # pyright: reportUnusedImport=none
 from .brand_jwt_auth import BrandJwtAuth
 from .models import BrandAuthConfig, BrandLogRecord
-from .request_brand_context import get_brand, get_request_brand, get_request_brand_alias
+from .request_brand_context import get_request_brand
```

Comment:
we might want to document this in the release log as breaking change and give an instruction on how to migrate